packages/block-editor/src/store/selectors.js

```diff
@@ -2214,15 +2214,24 @@ export const getAllowedBlocks = createSelector(
 			return;
 		}
 
-		return getBlockTypes().filter( ( blockType ) =>
+		const blockTypes = getBlockTypes().filter( ( blockType ) =>
 			canIncludeBlockTypeInInserter( state, blockType, rootClientId )
 		);
+		const hasReusableBlock =
```

Comment:
Why this can't be handled inside `canIncludeBlockTypeInInserter`? I'm checking the code, but if you have any insights, please share.

Reply:
The reusable block sets inserter to false and the selector returns early. See the why section, I’m happy clarify more details.

Reply:
You're right. Rushed into code and skipped the description, sorry about that 😄

Reply:
No a problem. I usually mess up my bug explanations, I’m glad it was okay this time 😁